allennlp/modules/token_embedders/bert_token_embedder.py

```diff
@@ -133,6 +134,9 @@ def forward(
             tokens from the first sentence should have type 0 and tokens from
             the second sentence should have type 1.  If you don't provide this
             (the default BertIndexer doesn't) then it's assumed to be all 0s.
+        mask : ``torch.LongTensor``, optional
+            This is not actually used here, but is needed for compatibility with other parts of the
+            code.
```

Comment:
Might be helpful to specify which other parts for people building analogues to this embedder in the future.

Reply:
Done.